lib/place_order.rb

```diff
@@ -0,0 +1,69 @@
+require './lib/menu.rb'
+
+class PlaceOrder
+
+  attr_accessor :order, :users_total, :total, :phone_number, :order_start
+
+  def initialize
+    @order = []
+    @users_total
+    @total
+    @phone_number
+    @order_start =[]
+  end
+
+  def ask_for_order
+    p "Please place your order in the form of order number, quantity, and start on a new line for each item E.g. 1, 7[return key]2, 4,[return key][return key]"
+  end
+
+  def multi_gets all_text=[]
+    while (text = gets) != "\n"
+      all_text << text.chomp
+    end
+    @order_start = all_text
+  end
+
+  def take_order
+      @order = @order_start
+  end
+
+  def ask_for_total
+    p "Please write your expected order total below e.g 12.36"
+  end
+
+  def take_user_total
+    @users_total =  gets.chomp.to_f
+  end
+
+  def calculate_total
+    # take first item, assume quantity 1
+    p @order
+    menu_item = Menu.new.nested_menu
+    if @order.length == 2
+      first_index = @order[0].to_i - 1
+      second_index = @order[1].to_f
+      @total = (menu_item[first_index][1] * second_index)
+    else
+      first_index = @order[0][0].to_i - 1
```

Comment:
Maybe you could use an iterative process here though it looks like this could be the simplest possible code for a menu of three items so maybe if the menu was larger this would make sense